genkit-tools/cli/src/commands/init.ts

```diff
@@ -193,7 +193,22 @@ export const init = new Command('init')
       process.exit(1);
     }
     const runtime = detectRuntime(process.cwd());
-    if (!supportedRuntimes.includes(runtime)) {
+    if (!runtime) {
+      logger.info('No runtime was detected in the current directory.');
+      if (
+        await confirm({
+          default: true,
+          message: 'Would you like to initialize a Node.js Genkit project?',
```

Comment:
What would it look like for Go?

Reply:
The intention was just to use Node.js as the default if nothing is detected since we don't have any support for Go here yet.